py-polars/tests/test_series.py

```diff
@@ -991,90 +1037,83 @@ def test_str_concat() -> None:
 
 def test_str_lengths() -> None:
     s = pl.Series(["messi", "ronaldo", None])
-    result = s.str.lengths()
     expected = pl.Series([5, 7, None], dtype=UInt32)
-    testing.assert_series_equal(result, expected)
+    verify_series_and_expr_api(s, expected, "str.lengths")
 
 
 def test_str_contains() -> None:
     s = pl.Series(["messi", "ronaldo", "ibrahimovic"])
-    result = s.str.contains("mes")
```

Comment:
This is an example of a test where we can now easily test both the Series and Expr api in one go

Reply:
Yes, very nice addition!